### PR TITLE
minoc: work around for bad Artifact.contentType and fits

### PR DESCRIPTION
**Pull request by pdowler**

### CODE REVIEW

File: minoc/src/intTest/java/org/opencadc/minoc/BasicOpsTest.java [hunk 1]
```diff
@@ -166,9 +167,21 @@ public void testPutGetUpdateHeadDelete() {
...
             Assert.assertEquals("Accepted", 202, post.getResponseCode());
+            Assert.assertEquals(newEncoding, post.getResponseHeader(HttpTransfer.CONTENT_ENCODING));
+            Assert.assertEquals(newType, post.getResponseHeader(HttpTransfer.CONTENT_TYPE));
+            
+            // invalid update
+            String invalidType = "bogus";
+            params.clear();
+            params.put("contentType", invalidType);
+            post = new HttpPost(artifactURL, params, false);
+            post.setDigest(computeChecksumURI(data));
```
Comment (andamian) [inline minoc/src/intTest/java/org/opencadc/minoc/BasicOpsTest.java:180]: Here too.

File: minoc/src/main/java/org/opencadc/minoc/PostAction.java [hunk 2]
```diff
@@ -68,12 +68,14 @@
...
 
 import ca.nrc.cadc.db.TransactionManager;
+import ca.nrc.cadc.net.ContentType;
 import ca.nrc.cadc.profiler.Profiler;
 import org.apache.log4j.Logger;
 import org.opencadc.inventory.Artifact;
 import org.opencadc.inventory.db.EntityNotFoundException;
 import org.opencadc.inventory.storage.PutTransaction;
 import org.opencadc.inventory.storage.StorageMetadata;
+import static org.opencadc.minoc.ArtifactAction.validateContentType;
```
Comment (andamian) [inline minoc/src/main/java/org/opencadc/minoc/PostAction.java]: Don't think you need to do this.

**Review by andamian — DISMISSED** [review]

File: minoc/src/intTest/java/org/opencadc/minoc/BasicOpsTest.java [hunk 3]
```diff
@@ -166,9 +167,21 @@ public void testPutGetUpdateHeadDelete() {
             post.setDigest(computeChecksumURI(data));
```
[the remark this answers is not in the document]
Comment (pdowler) [reply minoc/src/intTest/java/org/opencadc/minoc/BasicOpsTest.java:167]: hmm, that's old, let me check

File: minoc/src/main/java/org/opencadc/minoc/PostAction.java (hunk 2 above)
Comment (pdowler) [reply minoc/src/main/java/org/opencadc/minoc/PostAction.java]: yeah, noticed that (stupid IDE)

**Pull request merged**